docs/reference/stacker_cli.md

```diff
@@ -26,11 +29,12 @@
         stacker [global options] command [command options] [arguments...]
 
         VERSION:
-        stacker v0.40.1-e54a685 liblxc a330126b45c7c3b6fcf0f9ba6c1eda7bdb4e508a
+        stacker v1.0.0-rc4-8e267fc liblxc 727adc05220e9b8b594c5bc2124bef5b2e0abe7a
```

Comment:
maybe this should just be <version> and <liblxc commit hash>, since these won't directly match what's in the release?

Reply:
Your variables do not appear in the comment, but I think you mean this?
       stacker <version> liblxc <digest>

Reply:
yes, sorry!